terraform/dev/main.tf

```diff
@@ -0,0 +1,18 @@
+# Provision Intel Optimized Azure MySql server
+module "optimized-mysql-server" {
+  source              = "intel/azure-mysql_flexible_server/intel"
+  resource_group_name = "terraform-example-app-rg"
+  db_server_name      = "dev-intel-example-app01"
+  db_password         = var.db_password
+  tags = {
+    Owner    = "Lucas.Melo@intel.com"
```

Comment:
Wouldn't a more generic email account be better here?

Reply:
This is a "specific application", so this would be me(Lucas) deploying a "real app" to an Intel Subscription. 
This is not a generic module anymore. 

Makes sense?

Reply:
Your question is good because if it's not clear, I need to make it clearer...

Reply:
Sure.